tests/utils/test_time.py

```diff
@@ -0,0 +1,48 @@
+import asyncio
+from datetime import datetime, timezone
+from unittest.mock import patch
+
+import pytest
+from dateutil.relativedelta import relativedelta
+
+from bot.utils import time
+from tests.helpers import AsyncMock
+
+
+@pytest.mark.parametrize(
+    ('delta', 'precision', 'max_units', 'expected'),
+    (
+        (relativedelta(days=2), 'seconds', 1, '2 days'),
+        (relativedelta(days=2, hours=2), 'seconds', 2, '2 days and 2 hours'),
+        (relativedelta(days=2, hours=2), 'seconds', 1, '2 days'),
+        (relativedelta(days=2, hours=2), 'days', 2, '2 days'),
+    )
+)
+def test_humanize_delta(
+        delta: relativedelta,
+        precision: str,
+        max_units: int,
+        expected: str
+):
+    assert time.humanize_delta(delta, precision, max_units) == expected
+
+
+@pytest.mark.parametrize(
+    ('stamp', 'expected'),
+    (
+        ('Sun, 15 Sep 2019 12:00:00 GMT', datetime(2019, 9, 15, 12, 0, 0, tzinfo=timezone.utc)),
```

Comment:
Parameterisation seems unnecessary when there is only one.

Reply:
Yes, but:

- If we want to add a new test case, we only need to add a new line here
- It's consistent with the other test function

Reply:
What other test cases could be added? This seems too simple to warrant that, is all.

Reply:
That's hard to tell now. But if we ever find something here that requires adding a new test case (e.g. some input causing an error), then this would be simpler to modify